<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Файлоформирователь</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icons8-robot-2-64.png</normaloff>:/img/icons8-robot-2-64.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="speedCBX">
     <property name="text">
      <string>Запись на максимальной скорости</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="speedLBL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Скорость записи, Мб/c</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="speedSBX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="fileNameLED">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="3">
    <widget class="QSlider" name="speedHSL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="fileNameCBX">
     <property name="text">
      <string>Случайное имя файла</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>6</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startPBN">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Начать запись</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="fileNameLBL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Имя создаваемого файла</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="dirLED">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirPBN">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="dirLBL">
     <property name="text">
      <string>Каталог для создания файла</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>14</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="sizeLBL">
     <property name="text">
      <string>Размер файла, Мб</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="sizeSBX">
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSlider" name="sizeHSL">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileNameCBX</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNameLBL</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileNameCBX</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNameLED</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedCBX</sender>
   <signal>toggled(bool)</signal>
   <receiver>speedSBX</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedCBX</sender>
   <signal>toggled(bool)</signal>
   <receiver>speedHSL</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedCBX</sender>
   <signal>toggled(bool)</signal>
   <receiver>speedLBL</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSBX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sizeHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sizeSBX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSBX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSBX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
